.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,7 +11,8 @@ Applicable helpers:
 - [ ] TestCafe
 
 - Description of this PR, which problem it solves
-- A link to the corresponding issue (if applicable).
+- Fixes #issueId (if applicable).
+- Resolves #issueId (if applicable).
```

Comment:
Why do we ned both of them, if all just aliases
https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords#about-issue-references

Reply:
Initially, I think of putting both so anyone can pick one of them to use. But yeah, will update it.